src/api/app/helpers/webui/comments_helper.rb

```diff
@@ -27,21 +31,40 @@ def roles_for_package(comment)
 
   def roles_for_request(comment)
     roles = []
-    roles << 'Submitter' if comment.commentable.creator == comment.user.login
-    comment.commentable.bs_request_actions.inject(roles) do |acc, action|
-      acc += Relationship
-             .joins(:role)
-             .joins(:project)
-             .where(projects: { name: action.target_project })
-             .where(user: comment.user)
-             .pluck('roles.title')
-
-      acc + Relationship
-            .joins(:role)
-            .joins(:package)
-            .where(packages: { name: action.target_package })
-            .where(user: comment.user)
-            .pluck('roles.title')
+    roles << 'author' if comment.commentable.creator == comment.user.login
+    roles << 'reviewer' if comment.commentable.reviews.pluck(:user_id).include?(comment.user.id) ||
```

Comment:
I think you can just do `comment.commentable.reviews.map(&:user).include(comment.user)`.

Reply:
done